tests/integration/update.bats

```diff
@@ -133,7 +136,7 @@ EOF
         runc update test_update --memory-swap -1
         [ "$status" -eq 0 ]
         check_cgroup_value "$MEM_SWAP" $SYSTEM_MEM
-        check_systemd_value "$SD_MEM_SWAP" $SD_UNLIMITED
+        check_systemd_value "$SD_MEM_SWAP" $SD_UNLIMITED || check_systemd_value "$SD_MEM_SWAP" $SD_UNLIMITED_LEGACY
```

Comment:
This looks bad :(

Can we add a runtime check and set `SD_UNLIMITED` to the expected value?

Something like

```bash
SD_UNLIMITED="infinity"
# 
SD_VERSION=$(systemctl --version | awk '{print $2; exit}'
if [ "$SD_VERSION -lt 227 ]; then
   SD_UNLIMITED="18446744073709551615"
fi
```

Reply:
updated